src/css/common/vars/z-index.scss

```diff
@@ -1 +1,12 @@
-$tooltip-z-index: 10000;
+
+// sounds like ridiculously large number, but it's necessery
```

Comment:
Nice comment :)
It could be a little bit more explicit like "there are sites that use 100000 z-index", but I guess it's more or less clear.